image-layout.md

```diff
@@ -156,4 +161,48 @@ $ cat ./blobs/sha256/e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7f
 [tar stream]
 ```
 
+## index.json
+
+This OPTIONAL file is for use indexing references and elements of the image-layout, where walking the `refs` and `blobs` is sub-optimal.
```

Comment:
> For the name, that is why I made up the example [`org.opencontainers.ref.name`]

That works as a ref-name replacement, but if we actually want to use it (e.g. in image-tools), I think we want to define it [here][1].

[1]: https://github.com/opencontainers/image-spec/blob/72a840ec12140beaf29c043826a3d701d0820ee7/annotations.md#pre-defined-annotation-keys